Utility functions/plotBarIflearned.m

```diff
@@ -0,0 +1,49 @@
+function [p,Ls,nLs,All] = plotBarIflearned(Ls,nLs)
```

Comment:
the name of this function is not clear to me. Would the "plotLearnersWithNonlearners" better?